src/search.ts

```diff
@@ -78,4 +78,9 @@ export interface Search {
      * See https://www.hl7.org/fhir/capabilitystatement.html
      */
     getCapabilities(): Promise<SearchCapabilityStatement>;
+    /**
+     * Verify that a search string use as criteria in Subscription resource is valid
+     * and does not use _revinclude, _include, _sort, _count and chained parameter
```

Comment:
I'd remove this line. It's a bit too implementation specific. e.g.. our implementation does not support chained parameters since that would be too expensive with our DDB streams approach. Other implementations could in theory support them.